ai-saas/src/server/controllers/agents.controller.ts

```
@@ -15,9 +16,8 @@ export const getAgents = async (req: Request, res: Response) => {
 
 export const createAgents = async (req: Request, res: Response) => {
   try {
-
   } catch (error) {
     console.error(error);
     res.status(500).json({ message: "Failed to create agent" });
   }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)